recipe/meta.yaml

```diff
@@ -51,8 +51,20 @@ test:
     - sqlalchemy.util
   requires:
     - pip
+    - pytest >=7,<8.2
+    - mypy >= 1.7.0
+    - sqlite
+  source_files:
+    - pyproject.toml
+    - setup.cfg
+    - examples
+    - lib
+    - test
   commands:
     - pip check
+    # Error with test teardown closing sqlite database on python > 3.10 in a conda env.
+    - pytest -v test -m "not backend" -k "not aaa_profiling"  # [py<=310]
+    - pytest -v test -m "backend" -k "not aaa_profiling"
```

Comment:
Could you try this command https://github.com/sqlalchemy/sqlalchemy/blob/rel_2_0_33/pyproject.toml#L83? But add `pytest-xdist` to `test/requires`

Reply:
During my testing of the issue I did try disabling the timing intensive tests as well as the memory intensive ones. I also tested with and without `pytest-xdist`. Apparently I never tested skipping only the memory tests while `pytest-xdist` was installed because that seems to have fixed the issue.

Reply:
Well, it fixed them on a few platforms. As Dan said, I think the working test coverage we're getting for now will be sufficient. Perhaps we can take another look in a few versions and see if anything has changed.